<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealTimeData</class>
 <widget class="QDialog" name="RealTimeData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="inputLayout">
     <item>
      <widget class="QComboBox" name="deviceTypeCombo">
       <item>
        <property name="text">
         <string>选择设备类型</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="deviceIDCombo">
       <item>
        <property name="text">
         <string>选择设备ID</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>加载数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateChartButton">
       <property name="text">
        <string>生成折线图</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>数据ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数值</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
